### PR TITLE
Implement LHCB_WENU_8TEV

### PR DESCRIPTION
**Pull request by niclaurenti**

### CODE REVIEW

File: buildmaster/LHCB_WENU_8TEV/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          M2: {description: "W boson mass", label: "$M^2$", units: "$GeV^2$"}
+          sqrts: {description: "center of mass energy", label: r"$\sqrt s$", units: "$GeV$"}
+        file: kinematics_A.yaml
+      data_central: data_A.yaml
+      data_uncertainties:
+        - uncertainties_A.yaml
+      theory: 
+        FK_tables:
+          - - TODO
+        operation: 'null'
```
Comment (Radonirinaunimi) [inline buildmaster/LHCB_WENU_8TEV/metadata.yaml:73]: If FK tables are not available, then you can for the time being remove this entry.
```suggestion

```

**Review by Radonirinaunimi — COMMENTED** [review]
Thanks a lot @niclaurenti for this! I know that this is still in a draft mode but since we discussed this briefly I added some comments below.

File: buildmaster/LHCB_WENU_8TEV/filter.py [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+        source of uncertainties
+
+    """
+    errors = hepdata["dependent_variables"][0]["values"]
+
+    stat, sys_uncorr, sys_beam = [], [], []
+    for idx in bin_index:
+        stat.append(errors[idx]["errors"][0]["symerror"])
+        sys_uncorr.append(errors[idx]["errors"][1]["symerror"])
+        sys_beam.append(errors[idx]["errors"][2]["symerror"])
```
[the remark this answers is not in the document]
Comment (niclaurenti) [reply buildmaster/LHCB_WENU_8TEV/filter.py:130]: from this line
https://github.com/NNPDF/nnpdf/blob/f6b6c99bc28d1c8bdbd953d149feee79e91e1a00/buildmaster/filters/LHCB_WENU_8TEV.cc#L60
I assumed that the values stored in the rawdata folder (that is exactly what I implemented) are the absolute uncertainties

[the remark this answers is not in the document]
Comment (Radonirinaunimi) [reply buildmaster/LHCB_WENU_8TEV/filter.py:130]: You are absolutely right! I was comparing to a different table.

**Review by Radonirinaunimi — CHANGES_REQUESTED** [review]
Thanks again @niclaurenti for this! I've added a couple of corrections below.

I suggest to rename the dataset to be `LHCB_WPWM_8TEV_DILEPTON` in order to be confirm with the new naming conventions. Also, you should add the mapping from the old to the new name in `dataset_names.yml`:
```yaml
LHCB_WENU_8TEV_R: LHCB_WPWM_8TEV_DILEPTON
```

**Comment by scarlehoff** [comment]
Missing: producing the grids and the fktables for these datasets

**Pull request closed without merging**